index.php

```diff
@@ -0,0 +1,135 @@
+        </header>
+
+        <main>
+            <section>
+                <header>
+                    <h2 id="rules">
+                        <a href="#rules">Rules</a>
+                    </h2>
+                </header>
+
+                <p>You can submit a Ubuntu or Windows desktop (x64) application which adheres to the following conditions:</p>
+
+                <ol>
+                    <li>Excluding readme and copyright attribution, it is under 4 megabytes in size using the power of 2 system also known as <a href="https://en.wikipedia.org/wiki/Byte#Multiple-byte_units">Mebibytes (MiB)</a>.
+                        <ul>
+                            <li>It is the size of the unpacked project which is counted, not the compressed archive size.</li>
+                        </ul>
+                    </li>
+
+                    <li>It doesn't contain anything that violates any Canadian federal law.</li>
+                    <li>Isn't porn, doesn't have gratuitous nudity and explicit or implict sexual acts.</li>
+                    <li>It is a video game.</li>
+                    <li>You have a full month to do it, so it is expected it will be made during that month (may 2023).
+                        <ul>
+                            <li>Premade art, sound assets engine code are allowed.</li>
+                        </ul>
+                    </li>
+                    <li>It isn't a game mod or hack.</li>
+                    <li>It isn't an online multiplayer-only game.
+                        <ul>
+                            <li>Games with an online multiplayer component are fine as long as the main part of the game is entirely playable in solo. This is for judging reasons.</li>
+                            <li>Excluding file size, Judges are free to not include the multiplayer part of the game in their evaluation.</li>
+                            <li>If there's a dedicated server application, those files also count and must be included in the submission.</li>
+                        </ul>
+                    </li>
+                    <li>It doesn't stream nor download content from the internet.</li>
+                    <li>It is available for free for the duration of the judging period.</li>
+                    <li>It can be downloaded for the duration of the judging period.</li>
+                    <li>For clarification, a desktop application is an application that isn't a web application.
+                        <ul>
+                            <li>Something that embeds a web game into a binary desktop application, like electron or tauri is allowed. However, please keep in mind that some of these tools, like electron for example, will blow up the size of the executable way past the 4mb limit, so alternatives like tauri should be preferred.</li>
+                        </ul>
+                    </li>
+                </ol>
+
+                <a href="#">View Full Rules</a>
+            </section>
```

Comment:
these rules are out of date, see: https://gist.github.com/RealDoigt/c1ac8ef10c4012be6a5c3c1ab92f9df4

Since the rules are extensive. We can merge anyway and do them in a separate pr.